languages/rust/reference/merge-strategy.md

```diff
@@ -0,0 +1,34 @@
+# merging strategy
+
+The Rust track maintainers [have decided](https://github.com/exercism/v3/discussions/1725#discussion-7438) to write down our merging strategy to communicate expectations and encourage quality contributions.
+
+## general principles
+
+- we strive to accept _all_ contributions
+  - We'll bring contributions to a high standard of quality through collaborative, iterative review.
+  - We adopt a pessimistic merging approach. Think of maintainer reviews like the Rust compiler's feedback. The amount of feedback can feel overwhelming and frustrating. But we'll work with you to get it over the finish line! So keep up the good work.
+- One other maintainer should review a contributor's pull request.
+- At least one other maintainer should approve a pull request before merging it into the development branch.
+
+## pull request workflow
+
+### definitions
+
+| Term               |                                                                                               Meaning |
+| ------------------ | ----------------------------------------------------------------------------------------------------: |
+| Contributor        |                                                                 Anyone adding a new concept exercise. |
```

Comment:
```suggestion
| Contributor        |                                                                 Anyone submitting a pull request. |
```

There are lots of contributions other than adding concept exercises, and we want to encourage that.